### ISSUE TITLE
Export to COCO or Pascal VOC not returning full annotations

### ISSUE THREAD

**Issue by giuqoob** [heartexlabs/label-studio#4262]
**Describe the bug**
When I export annotations in JSON/CSV/TSV I get 96 annotations. When I export with COCO or Pascal VOC XML, I get all the annotations but only 46 images.

**To Reproduce**
1. Export files in TSV - excel has as many rows as there are annotations in label studio (Filter: Completed, is empty: no, Annotation results, not equal: [])
2. Export annotations to Pascal VOC or Coco. 
3. Open Annotations folder (Pascal VOC XML case), annotation data can be found.
4. Open images folder of export to see a different amount of images in the annotations, with part of the images that have annotation files missing.

Missing images include annotations I have just completed today.

**Expected behavior**
I'd expect exports to have all the annotations and respective images I have made and that are visible in Label studio.
**Screenshots**
![Image of view in Label studio about an annotated image](https://github.com/heartexlabs/label-studio/assets/90974572/c2a28c2c-7dd3-411b-a3e4-52bf3332a154)
![Annotation visible when using Export to TSV](https://github.com/heartexlabs/label-studio/assets/90974572/c6f110e1-8e3f-4fcb-923f-5b44ffb79b3e)
![Annotation visible in Annotations folder after Pascal VOC export](https://github.com/heartexlabs/label-studio/assets/90974572/caa16971-16f9-49cc-ab46-4041ed64bfab)
![Image not found in images folder](https://github.com/heartexlabs/label-studio/assets/90974572/1ee95450-ea05-4035-9332-8faa034cb0a7)

**Environment (please complete the following information):**
 - OS: Windows 11
 - Label Studio Version: 1.4 (1.7.3 install did not work)

**Additional context**
The images have been imported to label studio from different origins - some are from the same drive where Label studio is installed and run (C: drive), others are from a separate drive from the same machine (S: drive). Only images that were imported from C: drive are visible in the export results.

**Comment by giuqoob** [comment]
Maybe related: I've noticed that Pascal VOC XML Annotation filenames are encoded, e.g. original file name `test_Ä_test.jpg` would be converted to `test_%C3_test.jpg`. In the `label-studio/label-studio/media/upload` folder the images have the original name, but when they are exported it does look like the only files that were exported had no characters that had to be encoded.

**Comment by giuqoob** [comment]
I tried to upgrade to 1.7.3 and exported again - now all files are visible. The upgrade worked even though I got errors during the upgrade

`ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
pandas-flavor 0.3.0 requires lazy-loader==0.1rc2, but you have lazy-loader 0.2 which is incompatible.`
